packages/function-tree/src/index.js

```diff
@@ -191,6 +191,11 @@ export class FunctionTree extends EventEmitter {
     this.runTree.on = this.on.bind(this)
     this.runTree.once = this.once.bind(this)
     this.runTree.off = this.removeListener.bind(this)
+
+    // When not extended
+    if (!returnInstance) {
```

Comment:
just put `index.js` to the root and point to it from "main" in package,json.
I'll see what can be done here on my own.